birdhouse/components/twitcher/config/canarie-api/docker-compose-extra.yml

```diff
@@ -1,5 +1,6 @@
 version: "3.4"
 services:
-  proxy:
+  canarie-api: &canarie-volumes
     volumes:
     - ./components/twitcher/config/canarie-api/canarie_api_monitoring.py:${CANARIE_MONITORING_EXTRA_CONF_DIR}/twitcher_canarie_api_monitoring.py:ro
+  canarie-api-cron: *canarie-volumes
```

Comment:
Does this anchor work when all components are combined into one big compose file? 
Doesn't it get overridden by whichever definition was loaded first/last/whatever?

Reply:
Yes it works, docker compose parses yaml file individually first and then combines them into one big definition. Yaml anchors are applied at the file level.